### DISCUSSION TITLE
Downloading translation files includes strings not shown in the UI

### DISCUSSION THREAD

**Discussion by ghubz** [WeblateOrg/weblate#6313] [Q&A] [answered]
Hi,

When downloading a translations file it includes keys/strings that are not present in the UI. They must have been left from before updating the English/Source file.

Weblate 4.6.2

Any ideas how to fix this..? The source file has been changed quite a long time ago and there are no pending repo changes.

**Comment by nijel** [comment, marked as answer]
See https://docs.weblate.org/en/latest/devel/integration.html#updating-target-files

**Reply by ghubz** [reply]
Great thanks
